website/pages/en/querying/querying-by-subgraph-id-vs-deployment-id.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Subgraph ID vs Deployment ID
+---
+
+A subgraph is identified by a Subgraph ID, and each version of the subgraph is identified by a Deployment ID.
+
+When querying a subgraph, either ID can be used. However, there are some important differences between the two options.
+
+![](/img/subgraph-id-vs-deployment-id.png)
+
+## Subgraph ID
+
+The Subgraph ID is a unique identifier for a subgraph. It remains constant across all versions of a subgraph. When queries are made using the Subgraph ID, you are not specifying a version. This means that the version of the subgraph that responds to your query may not be the latest version.
+
+When you query using a Subgraph ID, The Graph will route your query to the latest synced and healthy version of the subgraph. This means that the version of the subgraph that responds to your query may not be the latest version, but it will be the most recent version that is synced and healthy.
```

Comment:
I feel like there's a bit of repetition between these two paragraphs. Maybe they should be merged?

```suggestion
The Subgraph ID is a unique identifier for a subgraph. It remains constant across all versions of a subgraph. When queries are made using the Subgraph ID, you are not specifying a version. This means that the version of the subgraph that responds to your query may not be the latest version. Rather, it will be the most recent version that is synced and healthy.
```